program-edit.php

```diff
@@ -59,22 +61,25 @@ function NewEditMovementRow(movementValue){ // adds a new section of movements
         var thisId = "divi" + (inputContainer.children.length); 
         select.id = thisId;
         select.name = "movement" + id;
+        select.classList.add('movement-input-box');
         newInputWrapper.appendChild(select); //adds the select element to the created div 
 
         var removeMovementButton = document.createElement('button');
-        removeMovementButton.textContent = 'Remove Movement';
+        removeMovementButton.textContent = '-Remove';
```

Comment:
This code feels very similar to `NewSetRow`. Do you think it'd be possible to combine them into one function, or at least make some helper functions that they both use?

Is there a reason why this one is in a PHP file but the other one is in a js file?